File: code-coaches/easy/skee-ball/skee_ball.rb
def can_buy_squirt_gun(score, squirt_gun_price)
  ratio = 12  # 12 points = 1 ticket
  tickets = score / ratio
  tickets >= squirt_gun_price
end

def main
  score = gets.chomp.to_i
  squirt_gun_price = gets.chomp.to_i

  if can_buy_squirt_gun(score, squirt_gun_price)
    puts "Buy it!"
  else
    puts "Try again"
  end
end

if __FILE__ == $0
  main
end

File: code-coaches/easy/skee-ball/skee_ball.cpp
#include <iostream>

bool can_buy_squirt_gun(int score, int squirt_gun_price)
{
    int ratio = 12; // 12 points = 1 ticket
	int tickets = score / ratio;
	return tickets >= squirt_gun_price;
}

int main()
{
	int score;
	std::cin >> score;

	int squirt_gun_price;
	std::cin >> squirt_gun_price;

    if (can_buy_squirt_gun(score, squirt_gun_price))
        std::cout << "Buy it!";
	else
		std::cout << "Try again";

	return 0;
}